prow/plugins/lifecycle/lifecycle.go

```diff
@@ -47,14 +47,14 @@ func help(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.Plu
 		Usage:       "/close",
 		Description: "Closes an issue or PR.",
 		Featured:    false,
-		WhoCanUse:   "Authors and assignees can triggers this command.",
+		WhoCanUse:   "For active issues or PRs, only authors and collaborators on the repository can trigger this command. For stale and rotten issues, anyone can trigger this command.",
```

Comment:
> also didn't realize anyone could close stale/rotten issues. Not sure how I feel about this

I was surprised as well.

>  Basically this allows anyone to close something by just /lifecycle stale and then /close in the next command.

Yeah, IMO it would be better to not have a check for active issues. All issues should only be allowed to be closed by a collaborator.

Reply:
I think this was added to allow @fejta-bot to close stale and rotten issues: https://github.com/kubernetes/test-infra/pull/5975.

Can we add @fejta-bot as a member/collaborator to the kubernetes org?

Reply:
I think we can follow up on this, and not hold this needed change up. Until then, I'd actually like to keep the close text the same as the reopen text, and not publicize this. Does that work @nikhita, @BenTheElder, @stevekuznetsov?

Reply:
>  Until then, I'd actually like to keep the close text the same as the reopen text, and not publicize this. 

Sgtm. Security by non-documentation. :)